### ISSUE TITLE
The data for Linear Evaluation Accuracy.

### ISSUE THREAD

**Issue by WenjinW** [DonkeyShot21/cassle#8]
Hi,

This is an exciting and enlightening work.

I wonder where the data for training the classifier come from, for linear evaluation accuracy.
The training data of the current task?

**Comment by DonkeyShot21** [comment]
Hi! It depends on the setting. For the normal linear eval experiments we used the same data used for the continual pre-training (all tasks), for or the semi-supervised experiments we used 1% or 10% of such data, and for the transfer experiments we used the domainnet real dataset.
